src/lib/storage/column_iterables/chunk_offset_mapping.cpp

```diff
@@ -18,4 +21,34 @@ ChunkOffsetsByChunkID split_pos_list_by_chunk_id(const PosList& pos_list) {
   return chunk_offsets_by_chunk_id;
 }
 
+std::optional<std::pair<ChunkID, ChunkOffsetsList>> to_chunk_id_and_chunk_offsets_list(const PosList& pos_list) {
```

Comment:
I love how the line comments in this functions increase it readability, even though they just rephrase the code below. Interesting...

Reply:
So is this good or bad? I’m not a fan of stating the obvious, but usually I’m asked to add more comments.

Reply:
Not sure tbh. I'd probably leave most of them out, but as I said they made things better for my first reading of this function. As part of this PR I'd just leave them in, we can discuss commenting strategies at another occasion.